test/util.js

```diff
@@ -169,6 +169,18 @@ return {
 	/**
 	 * Etc
 	 */
+	assertRuntimeBind: function( assert, formatter, runtimeKey, generatorString, runtimeArgsFn ) {
+		assert.ok( formatter );
+		assert.equal( typeof formatter, "function" );
+		assert.ok( "runtimeKey" in formatter );
+		assert.equal( formatter.runtimeKey, runtimeKey );
+		assert.ok( "generatorString" in formatter );
+		assert.equal( typeof formatter.generatorString, "function" );
+		assert.equal( formatter.generatorString(), generatorString );
+		assert.ok( "runtimeArgs" in formatter );
+		runtimeArgsFn( formatter.runtimeArgs );
```

Comment:
It looks like this avoids a temporary variable for the `assertRuntimeBind` called. I recommend refactoring that to actually use the temporary variable, since it'll hold the object that is actually being tested. Then drop the `runtimeArgsFn` argument and just call whatever other assertion with `formatter.runtimeArgs`. That removes a level of nesting in several tests along with a noop callback argument in many places.